style/ellipsis.scss

```diff
@@ -14,3 +14,16 @@
     padding: 8px 14px 10px;
   }
 }
+.#{$namespace}-ellipsis--multiple {
```

Comment:
Component style follows one-root principle (except Grid/Cell and Row/Column), so here should improve the style into only one root scope.

And you may run `pnpm run lint:style` to format the codes.

Reply:
thx! I will  follow the specifications and solve these problems in a day or two.